src/Tribe/Query.php

```diff
@@ -149,16 +149,23 @@ public static function parse_query( $query ) {
 			}
 
 			// never allow 404 on month view
-			if ( $query->is_main_query() && $query->get( 'eventDisplay' ) == 'month' && ! $query->is_tax && ! $query->tribe_is_event_category ) {
+			if (
+				$query->is_main_query()
+				&& 'month' === $query->get( 'eventDisplay' )
+				&& ! $query->is_tax
+				&& ! $query->tribe_is_event_category
+			) {
 				$query->is_post_type_archive = true;
 				$query->queried_object       = get_post_type_object( Tribe__Events__Main::POSTTYPE );
 				$query->queried_object_id    = 0;
 			}
 
-			// check if is_event_query === true and hook filter
-			if ( $query->tribe_is_event_query ) {
+			$events_as_front_page = tribe_get_option( 'front_page_event_archive', false );
```

Comment:
Do we need this/can we kill this line? Not sure I see any places where we reference this var.

Reply:
Woops...seems like we don't. Let me update it.